src/maps/russia.ru.js

```diff
@@ -0,0 +1,347 @@
+export default {
+  label: 'Россия',
+  viewBox: '0 0 990 593',
+  locations:[
+    {
+      name: 'Республика Карелия',
+      id: '10',
```

Comment:
Could you use semantic ids?
For example: `ca` for California, `ny` for New York...